tests/ffi/qm_nested/test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../common/prepare.sh
+
+# Declare global variables and initialize to 0
+NESTED_NAME=0
+NESTED_STATUS=0
+
+# Create nested container fedora inside the QM.
+# Function to create and start nested container fedora, while executing podman commands from inside the QM container.
+# Extract nested container's valuable information and store it inside the global variables for later use.
+# Globals:
+#  NESTED_NAME: The nested container name in the QM container.
+#  NESTED_STATUS: The nested container status in the QM container.
+create_nested() {
```

Comment:
Looking the files we have already, in the FFI dir, looks like the `--net host` might work actually. 
How about you use the common.sh created in the ffi ? The difference is that it's using `centos-sig-automotive/ffi-tools` image instead of fedora.  (in the end, this is what @Yarboa  is looking to for, re-use logics/code): 
https://github.com/containers/qm/blob/main/tests/ffi/deny_sched_setattr/test.sh#L1

For example, add the following in the create_nested() function:
```
#  Load previous functions already creating from prepare.sh
. ../common/prepare.sh

# init_ffi() uses functions from common.sh to init the env before tests.
init_ffi() {
    disk_cleanup 
    prepare_test
    reload_config
}

# create the nested container <Add a better description>
create_nested() {
   prepare_images
   local nested_container_name="ffi-qm"
   run_container_in_qm "${nested_container_name}"
   NESTED_NAME=$(podman exec -it qm bash -c "podman inspect "${nested_container_name}" --format '{{.Name}}'")
   echo "Container name is: ${NESTED_NAME}"
   NESTED_STATUS=$(podman exec -it qm bash -c "podman inspect --format='{{.State.Status}}' ${nested_container_name}")
   echo "Container $NESTED_NAME status is: ${NESTED_STATUS}"
}
```